settings.gradle.kts

```diff
@@ -3,11 +3,9 @@ plugins {
 }
 
 include("subclass",
-    "inline",
```

Comment:
While we need to delete the source files of this artifact, we do need to keep this module so it runs the tests. Please run the code coverage locally to see what it is doing, as it is reporting missing usage.

I recommend renaming this module to `inlineTest`. Then, we need to make sure we don't accidentally publish it, but iirc by appending Test to it, it works. I can double check that later.

Reply:
Oops, forgot about that. I checked the coverage locally and the html report doesnt' show decimals so I assumed 85% was it.
Now after I readded the renamed project it is at 86%.